GameOfThrones/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@ViewBag.Title - My ASP.NET Application</title>
```

Comment:
Not major but you could update the boilerplate text here with your application name

Reply:
Updated with my application's name